workspace/test.js

```diff
@@ -1,13 +1,9 @@
-import { assert, config as chaiConfig } from "chai";
-chaiConfig.truncateThreshold = 0;
+import { assert, config, toIntWith, compile, solution } from "./lc-test.js";
```

Comment:
Does `./lc-test.js` also have to be local? (I suppose so, for the same reason as `./files.js`)

Reply:
If you want, we can move `import.meta.url` to `test.js` instead:

```javascript
export const getSolution = (base) => readFileSync(new URL("./solution.lc", base), {encoding: "utf8"});
// or general
export const getRelative = (base, file) => readFileSync(new URL(file, base), {encoding: "utf8"});
```

```javascript
import { getSolution, getRelative } from "@codewars/lambda-calculus/test";

const { counter } = compile(getSolution(import.meta.url));

const { counter } = compile(getRelative(import.meta.url, "./solution.lc"));
```

Reply:
Yeah, `files.js` doesn't exist anymore.